specs/eventing/data-plane.md

```diff
@@ -1,125 +1,183 @@
+[binary](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#31-binary-content-mode)
+and
+[structured](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#32-structured-content-mode)
+content modes of the CloudEvents HTTP binding. Senders which do not advertise
+the ability to accept [reply events](#derived-reply-events) MAY implement only
+one content mode, as the recipient is not allowed to negotiate the content mode.
+Senders MUST support both cleartext (`http`) and TLS (`https`) URLs as event
+delivery destinations.
+
+### HTTP Verbs
+
+In the absence of specific delivery preferences, the sender MUST initiate
+delivery of the event to the recipient using the HTTP POST verb, using either
+the structured or binary encoding of the event (sender's choice). This delivery
+SHOULD be performed using the CloudEvents HTTP Binding, version 1.0.
+
+Senders MAY probe the recipient with an
+[HTTP OPTIONS request](https://tools.ietf.org/html/rfc7231#section-4.3.7); if
+implemented, the recipient MUST indicate support for the POST verb using the
+[`Allow` header](https://tools.ietf.org/html/rfc7231#section-7.4.1). Senders
+which receive an error when probing with HTTP OPTIONS SHOULD proceed using the
+HTTP POST mechanism.
+
+### Event Acknowledgement and Delivery Retry
+
+Event recipients MUST use the HTTP response code to indicate acceptance of an
+event. The recipient SHOULD NOT return a response accepting the event until it
+has handled the event (processed the event or stored it in stable storage). The
+following response codes are explicitly defined; event recipients MAY also
+respond with other response codes. A response code not in this table SHOULD be
+treated as a retriable error.
+
+| Response code | Meaning                                           | Retry | Delivery completed | Error |
+| ------------- | ------------------------------------------------- | ----- | ------------------ | ----- |
+| `1xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `200`         | [Accepted, event in reply](#derived-reply-events) | No    | Yes                | No    |
+| `202`         | Event accepted                                    | No    | Yes                | No    |
+| other `2xx`   | (Unspecified)                                     | No    | Yes                | No    |
+| `3xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `400`         | Unparsable event                                  | No    | No                 | Yes   |
+| `404`         | Endpoint does not exist                           | Yes   | No                 | Yes   |
+| `409`         | Conflict / Processing in progress                 | Yes   | No                 | Yes   |
+| `429`         | Too Many Requests / Overloaded                    | Yes   | No                 | Yes   |
```

Comment:
The spec should explicitly reference https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.3 as stated here: https://github.com/knative-sandbox/eventing-kafka/pull/666#issuecomment-854734214

The [CloudEvent Webhook spec](https://github.com/cloudevents/spec/blob/v1.0.1/http-webhook.md#22-delivery-response) says:

```
... The sender MUST observe the value of the Retry-After header and refrain from sending further requests until the indicated time.
```

With a proper reference to 7231 we should be good, right?